### ISSUE TITLE
`Invalid syntax. Got unexpected token 'as'` on multiple `... as ...` inside a with statement

### ISSUE THREAD

**Issue by Greesb** [charliermarsh/ruff#69]
Found this bug while testing ruff on my code, here is a simplified example to reproduce: 

`main.py`:
```python
with (
    open("test", "r") as f,
    open("test2", "r") as f2,
):
    print("hello")
```

ruff output:
```
[2022-09-01][14:00:00][ruff::pyproject][DEBUG] Found pyproject.toml at: pyproject.toml
[2022-09-01][14:00:00][ruff][DEBUG] Identified files to lint in: 3.25µs
[2022-09-01][14:00:00][ruff][ERROR] Failed to check main.py: invalid syntax. Got unexpected token 'as' at line 2 column 23
[2022-09-01][14:00:00][ruff][DEBUG] Checked files in: 174.625µs
Found 0 error(s).
```
